### ISSUE TITLE
Group state update time seems slow.

### ISSUE THREAD

**Issue by vandalon** [andreypopov/node-red-contrib-deconz#19]
When testing i noticed that group state is updated slowly, it takes up to 5 seconds before the group state reflects the real state. I ruled out that deconz sends out this state late with wsdump. When turning off a group deconz instantly sends out a update for that group:

```
< {"e":"changed","id":"41","r":"sensors","state":{"buttonevent":3002,"lastupdated":"2019-07-18T09:09:00"},"t":"event","uniqueid":"00:15:8d:00:02:38:7e:77-01-0006"}
< {"e":"changed","id":"12","r":"lights","state":{"alert":null,"bri":254,"colormode":"ct","ct":454,"on":false,"reachable":true},"t":"event","uniqueid":"00:15:8d:00:01:7d:91:ea-01"}
< {"e":"changed","id":"17","r":"lights","state":{"alert":null,"bri":254,"colormode":"ct","ct":500,"hue":12519,"on":false,"reachable":true,"sat":226,"xy":[0.5836,0.3893]},"t":"event","uniqueid":"00:17:88:01:00:dc:da:dc-0b"}
< {"e":"changed","id":"33","r":"lights","state":{"alert":null,"bri":1,"colormode":"ct","ct":454,"on":false,"reachable":true},"t":"event","uniqueid":"90:fd:9f:ff:fe:86:d2:20-01"}
< {"e":"changed","id":"9","r":"lights","state":{"alert":null,"bri":25,"colormode":"ct","ct":500,"hue":12519,"on":false,"reachable":true,"sat":226,"xy":[0.5268,0.4133]},"t":"event","uniqueid":"00:17:88:01:00:dc:ce:71-0b"}
< {"e":"changed","id":"4","r":"groups","state":{"all_on":false,"any_on":false},"t":"event"}
```

**Comment by vandalon** [comment]
it seems that groups are only updated with the node.refreshDiscoverInterval.
But I fail to understand how devices are update on change so far and how to apply that to groups :)

**Comment by vandalon** [comment]
#22

**Comment by vandalon** [comment]
Fixed in #23

**Comment by vandalon** [comment]
This was not an issue with groups, but with groups or devices existing in the get node and not in a in node. This has been fixed in #25

**Comment by andreypopov** [comment]
fixed since 0.11.0
